src/gui/widgets/FxLineLcdSpinBox.cpp

```diff
@@ -0,0 +1,62 @@
+/*
+ * FxLineLcdSpinBox.cpp - a specialization of LcdSpnBox for setting FX channels
+ *
+ * Copyright (c) 2004-2014 Tobias Doerffel <tobydox/at/users.sourceforge.net>
```

Comment:
Just realised this is just moving the already existing code into a new file, so we could maybe just extend the copyright